### DISCUSSION TITLE
Understanding the output of a jupyter lite build.

### DISCUSSION THREAD

**Discussion by savakarrohan** [jupyterlite/jupyterlite#843] [General]
## Goal 
I am trying to understand the output directory of a jupyter lite build. 
Although the comments on the `index.html` file is helpful, I'm just making sure my thoughts are appropriate.

## Question is it possible to change the location of the index.html file?
I tried changing the location of the index.html file from the default _output folder to a different folder as follows
```
|--_output
   |
   |-- _ All other jupyter lite build files _
|-- Voila
   |
   |-- index.html
```
That is I moved the root `index.html` file to the folder `Voila`

## What I changed in the `index.html` file.
Clearly there two locations where I have to update links in the index.html file, the `#jupyter-config-data` element and `config-utils.js` module.

I changed them to as follows:
```
<script id="jupyter-config-data" type="application/json" data-jupyter-lite-root="../_output/">
      {}
</script>
<script type="module">
      await import('../_output/config-utils.js');
</script>
```
Now this clearly doesn't work so well. Do you have any recommendations, or reading material where I can revise?

**Comment by bollwyvl** [comment]
> clearly doesn't work so well.

How does it fail? Any logs?

There are a lot of hard-coded relative paths... I don't think anybody has spent a lot of time trying to get different paths to line up.
